src/material/stepper/step-header.ts

```diff
@@ -99,8 +99,12 @@ export class MatStepHeader extends _MatStepHeaderMixinBase implements AfterViewI
   }
 
   /** Focuses the step header. */
-  focus() {
-    this._focusMonitor.focusVia(this._elementRef, 'program');
+  focus(origin?: FocusOrigin) {
```

Comment:
Should we add the `options` parameter here like the other `focus` methods. I don't feel strongly about it, it's just a suggestion.

Reply:
I agree, will update it!